collector/configuration/liverig.properties.md

```diff
@@ -68,7 +68,14 @@
 | `enable_witsml_tree_event` <br> **This Property was removed on Liverig 2.22.0**                                                                                                                                                                                                                                                                    | `boolean`  | `true`                               | 2.17.0 |           |                                                                                                          |                          |
 | `witsml_tree_event_period_seconds` <br> **This Property was removed on Liverig 2.22.0**                                                                                                                                                                                                                                                            |   `long`   | `600`                                | 2.17.0 |     s     |                                                                                                          |                          |
 | `wildcard_period_in_seconds` <br> Sets the Wildcard Period if the Request does not contain it.                                                                                                                                                                                                                                                     |   `int`    | `600`                                | 2.22.0 |     s     |                                                                                                          |                          |
-| `enable_revision`                                                                                                                                                                                                                                                                                                                                  | `boolean`  | `false`                              | 2.24.1 |           |                                                                                                          |                          |
+| `witsml_change_mechanism_feature` <br> Enables a mechanism for identifying change for data objects in a WITSML server | `boolean` | `false` | 5.17.x |||||
```

Comment:
It is important to mention that this new feature disables the realTime sync feature

Reply:
@efsh It is already mentioned in [new issue](https://github.com/intelie/gitbook-drilling/pull/55/commits/ffe55291d8b44b1b51c4ece3aeb829cd4b95a0bd#diff-10fe8dbb5f5ccc642a293b18881dd2ac37782b82f62411dacf6b165f8b60e3caR11) for #55